libraries/CurieSoftwareSerial/SoftwareSerial.cpp

```diff
@@ -94,14 +94,21 @@ bool SoftwareSerial::listen()
     bufferOverflow = false;
     _receive_buffer_head = _receive_buffer_tail = 0;
     active_object = this;
-    rxPin = _receivePin;
+    _rxPin = _receivePin;
+    rxIntraBitDelay = _rx_delay_intrabit;
+    rxCenteringDelay = _rx_delay_centering;
```

Comment:
If they are internal and only to be used inside the library, the convention is to precede name with _ character

Also if you made them static then scope is limited to the file they are defined in.